commands/deploy.go

```diff
@@ -558,12 +558,12 @@ func getRepositoryName(ff *common.FuncFileV20180708) (string, error) {
 	if len(parts) != 2 {
 		return "", fmt.Errorf("cannot parse image %s", ff.ImageNameV20180708())
 	}
-	pattern := regexp.MustCompile("(.*)\\.ocir\\.io/([^/]*)/(.*)")
+	pattern := regexp.MustCompile("(.*)ocir\\.([^/]*)/([^/]*)/(.*)")
```

Comment:
Why do you want to add another block of "([^/]*)" here?

Reply:
In OC1 the image name is of the format <region>.ocir.io/<tenancy>/<namespace>/<repository>:<tag>
 but higher realm image names does not follow this common pattern. For example in oc3 image name: "ocir.us-gov-ashburn-1.oci.oraclegovcloud.com/bmcdw/anuracha/hello-java:0.0.1". In oder to capture the part of the string between "ocir." and the next forward slash "/"  I have introduced "([^/]*)" block.